FirebaseStorageInternal/Tests/Integration/FIRStorageIntegrationTests.m

```diff
@@ -829,6 +845,24 @@ - (void)testListFilesAtRoot {
   [self waitForExpectations];
 }
 
+- (void)testCopyListResult {
+  XCTestExpectation *expectation = [self expectationWithDescription:@"testCopyListResult"];
+
+  FIRStorageReference *ref = [self.storage referenceWithPath:@""];
+
+  [ref listAllWithCompletion:^(FIRStorageListResult *_Nullable listResult,
+                               NSError *_Nullable error) {
+    XCTAssertNotNil(listResult);
+    XCTAssertNil(error);
+    XCTAssertNil(listResult.pageToken);
+    FIRStorageListResult *listResult2 = listResult.copy;
+    XCTAssertEqual(listResult.pageToken, listResult2.pageToken);
```

Comment:
Can these additional assertion be made?
```suggestion
    XCTAssertEqual(listResult.pageToken, listResult2.pageToken);
    XCTAssertNotEqual(listResult, listResult2);
    XCTAssertEqualObjects(listResult, listResult2);
```

Reply:
Only the first - since FIRStorageListResult doesn't and hasn't historically implemented `isEqual`